test/test_pkgconfig/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.16)
 project(GinkgoExportBuildWithPkgConfigTest LANGUAGES CXX)
 
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(GINKGO REQUIRED IMPORTED_TARGET ginkgo)
+pkg_check_modules(GINKGO REQUIRED IMPORTED_TARGET "ginkgo_${CMAKE_BUILD_TYPE}")
```

Comment:
Is there no imported target with just `ginkgo` now ?

Reply:
The `IMPORTED_TARGET` is just an option for PkgConfig, it has nothing to do with the package name. But I missed that we also installed the `ginkgo.pc`. I think that should be possible to fix.